maquetado/scss/dimensions.scss

```diff
@@ -0,0 +1,6 @@
+$book-image-height: 562px;
+$book-image-width: 398px;
+$book-image-margin: 1%;
+$book-info-width: 25%;
+$badge-height: 20%;
+$badge-width: 20%;
```

Comment:
Same, this are not reusable. Write this directly in your component. Use the same width as in the training invision